### PR TITLE
Add Trivy scan to GitHub Workflow

### PR DESCRIPTION
**Pull request by tunguyen9889**

### CODE REVIEW

**Review by lawliet89 — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -50,3 +50,53 @@ jobs:
           labels: ${{ steps.docker_meta.outputs.labels }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          outputs: type=docker,dest=/tmp/workload-standard-${{ github.sha }}.tar
+      - name: Upload artifact
```
Comment (sweatybridge) [inline .github/workflows/ci.yaml:54]:
```suggestion
      - name: Upload artefact
```

nitpick: we use British spelling internally.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -50,3 +50,53 @@ jobs:
...
+      matrix:
+        include:
+          - severity: "MEDIUM,HIGH"
+            exit-code: "0"
+          - severity: "CRITICAL"
+            exit-code: "1"
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Download artifact
```
Comment (sweatybridge) [inline .github/workflows/ci.yaml:75]:
```suggestion
      - name: Download artifact
        if: ${{ github.event_name == 'pull_request' }}
```

Seems like this step will only succeed for PR.

Comment (tunguyen9889) [reply .github/workflows/ci.yaml:75]: good catch

**Review by sweatybridge — APPROVED** [review]
LGTM overall.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (tunguyen9889) [reply .github/workflows/ci.yaml:54]: hmm, that will cause confusion since most of community documents or guideline are using this world `artifact`.

**Pull request merged**